File: JavaScript/1.js
function calc(a,b,op){
    if(op==="+"){
     return a+b;
    } else if(op==="-" ){
    return a-b;
    } else if(op==="*"){
      return a*b;
    } else if(op==="/"){return a/b;
    } else { return NaN; }
  }
  var a= parseFloat(prompt("Enter first number:"));
  var b= parseFloat(prompt("Enter second number:"));
  var op= prompt("Enter operator (+,-,*,/):");
  var result= calc(a,b,op);
  if(isNaN(result)){
    console.log("Invalid operator");
  }else { console.log("Result is "+ result); }
  for(var i=0;i<3;i++){
  console.log("Iteration "+i);
  }
  for(var j=0;j<2;j++){
    for(var k=0;k<2;k++){
    console.log("Nested "+j+","+k);
  }
  }
  console.log("Done");
  

File: Python/1.py
def calc(x,y,op):
 if op=="+":
  return x+y
 elif op=="-":
  return x-y
 elif op=="*":
  return x*y
 elif op=="/":
  return x/y
 else:
  return None
a=int(input("Enter first number:"))
b= int(input("Enter second number:"))
op=input("Enter operator (+,-,*,/):")
res= calc(a,b,op)
if res is None:
 print("Invalid operator")
else:
 print("Result is",res)
for i in range(3):
 print("Iteration",i)
for j in range(2):
 for k in range(2):
  print("Nested",j,k)
print("Done")
